assets/js/romo/dropdown.js

```diff
@@ -333,20 +334,20 @@ RomoDropdown.prototype.onResizeWindow = function(e) {
 }
 
 RomoDropdown.prototype.doPlacePopupElem = function() {
-  if (this.elem.parents('.romo-modal-popup').size() !== 0) {
-    this.popupElem.css({'position': 'fixed'});
+  if (Romo.parents(this.elem, '.romo-dropdown-popup').length !== 0) {
```

Comment:
@jcredding this needs to stay `.romo-modal-popup`.  This is doing a check to see if it is a dropdown in a modal.